.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+      actions: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache
+        uses: actions/cache@v3
```

Comment:
There is a v4 of this now

Reply:
Upgraded in [`dbacc8e`](https://github.com/flipstone/purescript-erumu/pull/2/commits/dbacc8e567c2d6ef534afde2d10b41f6e0b816e0).